src/templates/csr/capi-multiple-paidfor/index.svelte

```diff
@@ -0,0 +1,150 @@
+<script context="module" lang="ts">
+	export const cdn = 'https://i.guim.co.uk/img/media/';
+	export const api =
```

Comment:
Is this being used any more? It looks like this has moved to `capiMultiple.ts`

Reply:
Great spot, I'll remove it from this file